src/views/auth/Game.js

```diff
@@ -147,9 +148,20 @@ class Game extends React.Component {
       this.boardRef.current.updatePossibleTargetFields(possibleTargetFieldKeys)
     }
 
+    handlePlayerConnection(msg){
+      this.context.sockClient.send(`/app/game/${this.gameId}/game-end`, {});
+    }
+
+
```

Comment:
@edischmitz  Why is the `/app/game/${this.gameId}/game-end` end point is being hit on `componentDidMount()` of the `Game.js` React component?

Reply:
@iamsiddhantsahu It happens because I had to make a ConcurrentHashMap in the BackEnd to keep track of what Game I have to delete according to what user quits. This call Sends me the necessary info when the user arrive to the Game screen.I am working on making in a more elegant way but this works.

Reply:
@edischmitz Should I comment this line number `152` or is it needed?

Reply:
leave it for now, and if I find a better way I will take it out

Reply:
Okay. We can may be have a different end point for this or send to a new or existing topics. Committing and merging for now.